src/constants/cities.js

```diff
@@ -38,7 +38,7 @@ export default {
   },
   'Gold Coast - Tweed Heads': {
     id: 'gold-coast-tweed-heads',
-    name: 'Gold Coast-Tweed',
+    name: 'Gold Coast-Tweed Heads',
```

Comment:
Maybe make sure that whoever asked for this understands that it affects the labels on the chart axis too. We've gone backwards and forwards on this a few times.

Reply:
Yeah Nick was aware of the chart thing too.